src/actions/getJwtToken.ts

```diff
@@ -2,12 +2,51 @@ import { functions } from '../firebase/firebase'
 import { UID } from '../../types/uid'
 import { JWTToken } from '../../types/JWT'
 
+const RETRY = {
+    number: 4,
+    interval: 300,
+}
+
+async function sleep(intervalMillis: number) {
+    return new Promise((resolve) => {
+        setTimeout(function () {
+            resolve(true)
+        }, intervalMillis)
+    })
+}
+
+async function tryAtMost(promise, maxRetries, retryInterval) {
+    let retriesNumber = 0
+    while (retriesNumber <= maxRetries) {
+        try {
+            retriesNumber++
+            const result = await promise
+            return result
+        } catch (err) {
+            await sleep(RETRY.interval + RETRY.number * 1.3)
```

Comment:
in the last comment i used `^` rahter than multiply, which grealty increase the duration on retry